app/Controller/Component/UpdateComponent.php

```diff
@@ -232,6 +232,10 @@ public function updateDb()
         $db = ConnectionManager::getDataSource('default');
         $queries = [];
         foreach ($diffSchema as $table => $changes) {
+            // If we have actions (maybe we've removed the only action `drop`)
+            if (count($diffSchema[$table]) > 0) {
```

Comment:
why move this? We only need to alterSchema when we have dropped instructions asking to delete plugin tables

Reply:
I say it in description